.travis.yml

```diff
@@ -9,5 +9,5 @@ before_install:
   - docker build -t build_ref -f dockerImage .
 
 script: 
-  - docker run build_ref /bin/bash -c "mkdir build-clang-debug && cd build-clang-debug && cmake .. && make -j$(nproc) && ctest"
-  - docker run build_ref /bin/bash -c "mkdir build-gcc-release && cd build-gcc-release && cmake -DCMAKE_CXX_COMPILER=g++ -DCMAKE_BUILD_TYPE=Release .. && make -j$(nproc) && ctest"
+  - docker run build_ref /bin/bash -c "mkdir build-clang-debug && cd build-clang-debug && cmake .. && make -j$(nproc) && ctest --verbose"
```

Comment:
Are you sure that these two lines do different build? Are you sure that you ever build with clang?

Reply:
@malirod, not sure. Replaced by new implementation